### DISCUSSION TITLE
'Error adding document' no such index

### DISCUSSION THREAD

**Discussion by corycandia** [Security-Onion-Solutions/securityonion#5242] [Unsupported Versions]
Experts,

I haven't found any info on this yet, so here I am again.

Trying to test a pipeline change, I usually just select 'add document', 'Add a test document from an index'.  It was working in my manager search 2.3.61, not working in my manager 2.3.70.

Trying to add document:
![image](https://user-images.githubusercontent.com/88662234/130339137-b6081f5d-cb5d-4602-99b1-28900d053029.png)

Source Document:
![image](https://user-images.githubusercontent.com/88662234/130339141-fe0640b3-9782-490f-ac93-272d6c0465e4.png)

I noticed there are no SO indexes listed under stack management.  I know they are on the search node, but shouldn't they still be listed?
![image](https://user-images.githubusercontent.com/88662234/130339154-04a93047-e2b8-4bd8-ab96-5aa8b3f7baf2.png)

[Elastic docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.13/index-mgmt.html#index-mgm-req-permissions) list permissions, but the so-user-add doesn't seem to add users in a way that they show up in the GUI tool.

I'm not sure how to proceed as no one else seems to mention this anywhere.

**Comment by weslambert** [comment]
You'll likely need to prefix the index name, like:

$nodename:$index

Otherwise, it thinks you are attempting to add to an index on the manager (and the index won't exist there).

**Reply by corycandia** [reply]
I must still be doing something wrong?
![image](https://user-images.githubusercontent.com/88662234/131237149-8fed0432-11e0-4c4e-8ec7-16e3bf6189db.png)

The cluster shows connected, I can query the docs
